README.md

```diff
@@ -1,24 +1,55 @@
-# README
+[![Build Status](https://travis-ci.org/sul-dlss/vatican_exhibits.svg?branch=master)](https://travis-ci.org/sul-dlss/vatican_exhibits)
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+## Vatican Exhibits
 
-Things you may want to cover:
+Spotlight-based discovery application for the Vatican Library built by the Stanford Libraries. Built using:
 
-* Ruby version
+* [Spotlight](https://github.com/projectblacklight/spotlight)
+* [Mirador](https://github.com/projectmirador)
 
-* System dependencies
 
-* Configuration
+## Requirements
```

Comment:
I know that uploading masthead images etc. isn't the focus of this project, but perhaps it would be good to include ImageMagick as a requirement?

Reply:
Sure; should we put that inline or add a reference to the spotlight list (e.g. "and also the requirements for spotlight")?

Reply:
I'd go with inline just to be safe.